XamarinCommunityToolkit/Behaviors/Validators/TextCharactersValidationBehavior.shared.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+	public class TextCharactersValidationBehavior : TextValidationBehavior
+	{
+		List<Predicate<char>> characterPredicates;
+
+		public static readonly BindableProperty CharacterTypeProperty =
+			BindableProperty.Create(nameof(CharacterType), typeof(CharacterType), typeof(TextCharactersValidationBehavior), CharacterType.Any, propertyChanged: OnCharacterTypePropertyChanged);
+
+		public static readonly BindableProperty MinimumCharactersNumberProperty =
```

Comment:
I'm not a fan of this property name, it sounds a bit strange.

- `MinimumNumberOfCharacters`
- `MinimumCharacterCount`
- `MinCharacterCount`
- `MinLength` & `MaxLength`

Perhaps mimicing `MaxLength` as it already exists on Entry would be the way to go?

Reply:
TextCharactersValidationBehaviors extends TextValidationBehavior.

MaximumLenght and MinimumLength exist in TextValidationBehavior (They are used for checking entire string length)
But these properties (MinimumCharactersNumber and MaximumCharactersNumber) are used for checking if a string contains the necessary quantity characters of a certain type.

Reply:
MinimumCharacterCount looks good to me

Reply:
Updated